packages/codemods/README.md

```diff
@@ -0,0 +1,26 @@
+# @backstage/codemods
+
+A collection of codemods for use with Backstage projects.
+
+## Usage
+
+This package is a wrapper around [`jscodeshift`](https://github.com/facebook/jscodeshift) with some included transforms. The transforms can either be executed via the included CLI or directly via `jscodeshift`.
+
+To run the `core-imports` codemod towards all source files in a package, run the following:
```

Comment:
Is there any way for users to identify WHAT Codemods exist?  Doesn't seem to be like an `npx @backstage/codemods --list` command or any such yet?  Unless the intent is all code mods would be only documented here in the README as a quick way to list them?

Reply:
Actually just `npx @backstage/codemods` and it'll show a list, but plan is indeed to document them all in the README as it takes a bit of time to run that